CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Please enumerate **all user-facing** changes using format `<githib issue/pr numb
 
 ## 0.3.0
 
+* [#531](https://github.com/kroxylicious/kroxylicious/pull/531 Mock test client supports multi-RPC conversations with the server.
```

Comment:
again this makes it sound like a client that mocks things out not a stubbed broker.